system/scdoc/scdoc.info

```diff
@@ -1,8 +1,8 @@
 PRGNAM="scdoc"
 VERSION="1.11.1"
```

Comment:
Can you update the VERSION here to match please?

Reply:
Done, sorry for this